src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1852,19 +1852,44 @@ namespace DataHeatBalance {
     {
         // Members
         bool isOccupied;        // occupied during the current time step
+        Real64 NumOcc;          // number of occupants - used in calculating Vbz
         Real64 NumOccAccum;     // number of occupants accumulating for entire simulation
         Real64 NumOccAccumTime; // time that the number of occupants is accumulating to compute average
-        //  - zone time step
+        //  - zone time step [hrs]
         Real64 TotTimeOcc; // time occupied (and the mechanical ventilation volume is accumulating)
-        //  - system time step
-        Real64 MechVentVolTotal; // volume for mechanical ventilation of outside air for entire simulation
-        Real64 MechVentVolMin;   // a large number since finding minimum volume
-        Real64 InfilVolTotal;    // volume for infiltration of outside air for entire simulation
-        Real64 InfilVolMin;      // a large number since finding minimum volume
-        Real64 AFNInfilVolTotal; // volume for infiltration of outside air for entire simulation
-        Real64 AFNInfilVolMin;   // a large number since finding minimum volume
-        Real64 SimpVentVolTotal; // volume for simple 'ZoneVentilation' of outside air for entire simulation
-        Real64 SimpVentVolMin;   // a large number since finding minimum volume
+        //  - system time step [hrs]
+
+        // OA Reports - accumulated values
+        // All Vol variables are in m3
+        Real64 MechVentVolTotalOcc;       // volume for mechanical ventilation of outside air for entire simulation during occupied at current
```

Comment:
Rename some of the existing Outdoor Air Summary variables to help keep them separate from the new report. The old report is only for occupied hours, so add "Occ" to some names.

Reply:
OK, I think that is reasonable.